.github/workflows/integration_test_charm.yaml

```diff
@@ -113,9 +113,6 @@ jobs:
     # (In the UI, when this workflow is called with a matrix, GitHub will separate each matrix
     # combination and preserve job ordering within a matrix combination.)
     name: ${{ inputs.juju-agent-version || inputs.juju-snap-channel }} | ${{ inputs.architecture }} | Collect integration test groups
-    # Only run juju 3.6 tests on `schedule`
-    # Temporary while juju 3.6 is unstable (to avoid blocking PRs but collect data on nightly CI)
-    if: ${{ !startsWith(inputs.juju-snap-channel, '3.6/') || github.event_name == 'schedule'}}
```

Comment:
i think we still want to run juju 3.6/stable tests on a nightly basis for another week before running them on all PRs

Reply:
LGTM. IMHO, we should also stop testing Juju 3.4/3.5 and test only Juju 2.9 LTS + 3.6 LTS.

Reply:
@shayancanonical we have already run Juju 3.6 at night for a week and I didn't notice (new, 3.6 specific) issues there.

Reply:
while this may be true for mysql tests, postgres might face some heat with regards to 3.6 tests as there are consistent failures in schedule CI tests (might result in a block of PRs until these tests are stabilized). @taurus-forever approving, but please provide input if you still think that this change should go in

Reply:
This. All seems well, but let's wait

Reply:
Show failing 3.6 PostgreSQL tests to @marceloneppel .
Juju 3.6 LTS is officially released/ shipped, if we have issues with charm there - we need a fix asap. 
It it is a standard CI instability noise => backlog ticket to process normally.

We should test Juju 2.9 LTS and Juju 3.6 LTS. Other Juju versions can be removed from testing.